features/users/user_create.feature

```diff
@@ -77,8 +77,10 @@ Feature: Sign Up for a new account
 
   Scenario: The user should be able to create a new account with a valid email and password
     When I fill in the sign up form with valid data
+    Then I should see the page title "Create Account | Example Archive"
```

Comment:
We prefer to leave off the " | Example Archive" part for the tests so people who've changed the name of their dev environment don't run into test failures.